### ISSUE TITLE
Unable to authenticate with Kitsu

### ISSUE THREAD

**Issue by SimonSheehan** [Atelier-Shiori/hachidori#14]
I've attempted to connect Hachidori to my Kitsu account but it cannot seem to find my account. Initially I did not have a profile URL and was attempting to log in as "Simon". Upon realizing there were multiple people with this name, I tried to use the number generated to my profile URL to log in, and failed as well. I now have a proper URL there, but still cannot log in. I'm thinking at this point it is a bug with authentication. I've also attempted using my email, since apparently the Kitsu API looks for emails as the "username". Please let me know any other information or testing you may need.

I reported this as a bug to Kitsu but since I've successfully logged into the site, it doesn't seem to be an API bug, so we can only assume it is an issue with Hachidori at this point.

![screen shot 2018-03-15 at 5 03 21 pm](https://user-images.githubusercontent.com/716487/37492312-47e5374e-2877-11e8-84a0-4c050680dfc3.png)

**Comment by chikorita157** [comment]
I have changed how the user ID is retrieved in 2.2.3,. which I released two hours ago so it uses the OAuth token to retrieve the userid of the said credential instead of searching by name. I only realized today that the name filter is not the actual username.

The profile URL username should be used, but it seems that is not working on your end. Try the 2.2.3 build first. I'll leave this ticket open.
